gui/lib/Help.elm

```diff
@@ -124,6 +124,33 @@ view helpers model =
                     ]
                 ]
             , h2 [] [ text (helpers.t "Help Official Support") ]
+            , if model.status == Success then
+                p [ class "message--success" ]
+                    [ text (helpers.t "Help Your mail has been sent. We will try to respond to it really soon!") ]
+              else
+                Html.form [ class "send-mail-to-support" ]
+                    [ case model.status of
+                        Error error ->
+                            p [ class "message--error" ]
+                                [ text ("Error: " ++ error) ]
```

Comment:
Will this be a low-level error message? Should we show it to the user, or log it and display a more user-friendly message?